config/internal/apiserver/default/kfp_launcher_config.yaml.tmpl

```diff
@@ -1,5 +1,8 @@
 apiVersion: v1
 data:
+  {{ if .APIServer.CustomKfpLauncherConfigMap }}
+  {{.CustomKfpLauncherConfigMapData}}
```

Comment:
I see in your instructions: 

```yaml
```yaml
kind: ConfigMap
apiVersion: v1
metadata:
  name: my-custom-kfp-launcher
  namespace: dspa1
data:
  defaultPipelineRoot: 's3://some-path'
  greg: isCool
  hey: |
    - what: isUp
```

I didn't try it myself, but have you tried with a sample configmap that is similar in structure to examples provided here: https://github.com/opendatahub-io/data-science-pipelines-operator/pull/681 

I ask because, yamls with more tabs can sometimes cause formatting issues, so it's worth making sure they also get copied correctly.
```

Reply:
did you link the right thing?

I think I am doing what you're asking, with the pipe?

At any rate, by accessing the map[string]string from the ConfigMap type, and immediately converting that to json, I think we will avoid any whitespace weirdness that could happen. I'm not doing any string or spacing manipulation myself.

Reply:
sorry meant to link the docs: https://github.com/kubeflow/website/pull/3833 

yeah I think with the "- what" tab it's suggestive that there's no issue, if we can get someone to verify with an example that is more similar to the real structure would be better, but if you're confident here I will yield to your judgement

Reply:
no harm in adding a more exhaustive test! thanks :)

Reply:
:heavy_check_mark: